python/ray/train/tests/test_callbacks.py

```diff
@@ -44,6 +47,18 @@ def make_temp_dir():
         shutil.rmtree(tmpdir)
 
 
+make_temp_dir_2 = make_temp_dir
+
+
+@pytest.fixture
+def make_temp_file():
```

Comment:
I know the `make_temp_dir` fixture already existed before, but I think these can all be replaced by the pytest built-in fixtures https://docs.pytest.org/en/6.2.x/tmpdir.html.

Specifically for multiple tmpdirs, I think you can use the tmp_dir_factory fixture.

Reply:
Oh this is new to me, thanks for sharing!

Reply:
Updated to just use  `tmp_path`